extra-utils/mcfly/autobuild/postinst

```diff
@@ -0,0 +1,13 @@
+echo -e "
+mcfly will be loaded automatically the next time you start the bash shell, or if you are using the bash shell and loading it immediately, type:
```

Comment:
McFly will be loaded automatically the next time you enter the Bash shell. If you would like to load McFly for your current shell session, please input the following command: